maps/z3.dmm

```diff
@@ -4247,7 +4241,6 @@
 /area/space)
 "amd" = (
 /obj/grille/catwalk,
-/turf/space,
 /turf/simulated/floor/airless/plating/catwalk,
 /turf/simulated/floor/airless/plating/catwalk{
```

Comment:
this is still 2 stacked catwalk turfs with different dirs- does that do some visual thing we want?

Reply:
fixed, thank you for catching it